package.json

```diff
@@ -37,7 +37,7 @@
     "mobx": "^5.15.4",
     "mobx-react": "^6.2.2",
     "prop-types": "^15.6.2",
-    "query-string": "^8.1.0",
+    "query-string": "7.1.3",
```

Comment:
I had to downgrade `query-string` package, otherwise the whole project needs to be changed to `module`-type.